ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -54,6 +54,26 @@ RSkComponentTextInput::RSkComponentTextInput(const ShadowView &shadowView)
   sem_init(&jsUpdateSem,0,0);
   isKeyRepeateOn=false;
 }
+SkPaint RSkComponentTextInput::convertTextColor ( SharedColor textColor ) {
+  SkPaint paint;
+  float ratio = 255.9999;
+  auto color = colorComponentsFromColor(textColor);
+  paint.setColor(SkColorSetARGB( color.alpha * ratio,
+                                     color.red * ratio,
+                                     color.green * ratio,
+                                     color.blue * ratio));
+  paint.setAntiAlias(true);
+  return paint;
+}
+
+
+void RSkComponentTextInput::drawUnderline(SkCanvas *canvas,Rect frame,SharedColor underlineColorAndroid){
```

Comment:
it would be good , if this functionality goes to drawutils for reusability

Reply:
agreed